linera-execution/src/runtime.rs

```diff
@@ -946,7 +946,7 @@ where
             this.transaction_tracker
                 .replay_oracle_response(OracleResponse::Blob(blob_id))?;
         }
-        Ok(content.into_bytes().into_vec())
+        Ok(content.into_bytes().to_vec())
```

Comment:
Now we are always cloning the data here. :confused: 
So maybe it actually makes sense to use `Arc<Box<[u8]>>` after all? Then we could use `Arc::unwrap_or_clone` here.

But maybe _this_ method should return `Arc<[u8]>` (or `Arc<Box<[u8]>>`) anyway? There's no reason to move out of the `Arc` yet.